packages/api/tests/lib/reconciliationReport/test-DbGranuleSearchQueues.js

```diff
@@ -0,0 +1,101 @@
+'use strict';
+
+const test = require('ava');
+const range = require('lodash/range');
+const { randomId } = require('@cumulus/common/test-utils');
+const { DbGranuleSearchQueues } = require('../../../lib/reconciliationReport/DbGranuleSearchQueues');
+
+class StubGranuleSearchQueue {
+  constructor(items) {
+    this.items = items || [];
```

Comment:
this could be a shallow copy of items.
```suggestion
    this.items = [...items] || [];
```
And then you could pass in a list directly, rather than using array.slice() later. 

Or I think it can.